extensions/authclient/clients/GoogleOpenId.php

```diff
@@ -1,91 +0,0 @@
-<?php
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
-
-namespace yii\authclient\clients;
-
-use yii\authclient\OpenId;
-
-/**
- * GoogleOpenId allows authentication via Google OpenId.
- * Unlike Google OAuth you do not need to register your application anywhere in order to use Google OpenId.
- *
- * Example application configuration:
- *
- * ~~~
- * 'components' => [
- *     'authClientCollection' => [
- *         'class' => 'yii\authclient\Collection',
- *         'clients' => [
- *             'google' => [
- *                 'class' => 'yii\authclient\clients\GoogleOpenId'
- *             ],
- *         ],
- *     ]
- *     ...
- * ]
- * ~~~
- *
- * @author Paul Klimov <klimov.paul@gmail.com>
- * @since 2.0
- */
-class GoogleOpenId extends OpenId
```

Comment:
OK, I can restore the file.
When it should be safe to be removed then?
I don't like the idea to keep it for long, because it may confuse newcomers.